File: Day-02-Strings/07-Anagram-Check/Solution.java
import java.util.*;
class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String s1 = sc.next();
        String s2 = sc.next();

        int[] freq = new int[26];

        for(char c : s1.toCharArray())
            freq[c - 'a']++;

        for(char c : s2.toCharArray())
            freq[c - 'a']--;

        for(int i = 0; i < 26; i++) {
            if(freq[i] != 0) {
                System.out.print("false");
                return;
            }
        }

        System.out.print("true");
    }
}

File: Day-02-Strings/07-Anagram-Check/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    string s1, s2;
    cin >> s1 >> s2;

    int freq[26] = {0};

    for(char c : s1)
        freq[c - 'a']++;

    for(char c : s2)
        freq[c - 'a']--;

    for(int i = 0; i < 26; i++) {
        if(freq[i] != 0) {
            cout << "false";
            return 0;
        }
    }
    cout << "true";
}